src/app/app.component.ts

```diff
@@ -104,7 +106,7 @@ export class AppComponent implements OnInit, OnDestroy {
   getSeriesPodcastDistribution(podcast: PodcastModel) {
     podcast.doc.followItems('prx:distributions').subscribe((distros: HalDoc[]) => {
       const podcasts = distros.filter((doc => doc['kind'] === 'podcast'));
-      if (podcasts && podcasts.length > 0) {
+      if (podcasts && podcasts.length > 0 && podcasts[0]['url']) {
```

Comment:
Found some of these without a URL, huh?  I hope not in prod!  Haven't seen that bug for awhile now.

Reply:
yeah... got an error on the split, didn't really investigate 🤷‍♀️